buildSrc/src/main/kotlin/Java9Modularity.kt

```diff
@@ -0,0 +1,115 @@
+import org.gradle.api.JavaVersion
+import org.gradle.api.Project
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.api.tasks.compile.AbstractCompile
+import org.gradle.api.tasks.compile.JavaCompile
+import org.gradle.kotlin.dsl.attributes
+import org.gradle.kotlin.dsl.findByType
+import org.gradle.kotlin.dsl.getByName
+import org.gradle.kotlin.dsl.register
+import org.gradle.util.GUtil.toCamelCase
+import org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension
+import org.jetbrains.kotlin.gradle.plugin.KotlinCompilation
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinWithJavaTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.pm20.targets
+import org.jetbrains.kotlin.gradle.targets.jvm.KotlinJvmTarget
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.PrintStream
+import java.lang.module.ModuleDescriptor
+import java.util.spi.ToolProvider
+
+object Java9Modularity {
+
+    @JvmStatic
+    fun Project.configureJava9ModuleInfo(multiRelease: Boolean = false) {
```

Comment:
Shouldn't it be `true` by default? You don't pass `multiRelease` explicitly anywhere

Reply:
Correct, changed that.